packages/react/src/Banner/Banner.tsx

```diff
@@ -474,22 +475,28 @@ export function BannerActions({primaryAction, secondaryAction}: BannerActionsPro
   )
 }
 
-export type BannerPrimaryActionProps = Omit<React.ComponentPropsWithoutRef<typeof Button>, 'variant'>
+export type BannerPrimaryActionProps = Omit<ButtonProps, 'variant'>
 
-export function BannerPrimaryAction({children, className, ...rest}: BannerPrimaryActionProps) {
+const BannerPrimaryAction = forwardRef(({children, className, ...rest}, forwardedRef) => {
   return (
-    <Button className={clsx('BannerPrimaryAction', className)} variant="default" {...rest}>
+    <Button ref={forwardedRef} className={clsx('BannerPrimaryAction', className)} variant="default" {...rest}>
       {children}
     </Button>
   )
-}
+}) as PolymorphicForwardRefComponent<'button', BannerPrimaryActionProps>
```

Comment:
I don't think we necessarily care about forwarding a ref here but it's the only way I found to get the type-checking to pass (mimicked [Button](https://github.com/primer/react/blob/main/packages/react/src/Button/Button.tsx#L26)). Open to suggestions

Reply:
It makes sense to me, I'd do the same but yeah open to be convinced otherwise 😅